gemlite/core.py

```diff
@@ -178,6 +178,9 @@ def eval_time(fct, params, warmup=25, rep=200, fast_flush=True, return_mode="min
     ("BF16", "GEMM"): gemm_A16fWnO16f_int32packing,
 }
 
+def get_closest_m(M):
```

Comment:
You don't need to put it in `core.py` since it only selects the faster implementation (gemm or gemv)

Reply:
This was to limit the unique `_signature`'s and avoid calling `self.warmup` multiple times.